flipper_protobuf.py

```diff
@@ -4,7 +4,7 @@
 from google.protobuf.internal.encoder import _VarintBytes
 from numpy import mat
 
-from .flipperzero_protobuf_compiled import flipper_pb2, system_pb2, gui_pb2
+from .flipperzero_protobuf_compiled import flipper_pb2, system_pb2, gui_pb2, gpio_pb2
```

Comment:
For now, you need to run rebuild.sh, it's not automated yet.

Reply:
Yeah, I'm aware, I have a PR on the protobuf submodule but I can't bump the version in dependent projects, so I was conflicted on how to submit this (and all other PRs that depend on this change).

Locally I symlinked and used `rebuild.sh` to generate the files.

When/If the protobuf PR is merged, I'll update all PRs that depend on this change